docs/UserGuide.md

```diff
@@ -475,96 +475,130 @@ Note: Calling mark or unmark would only prompt an error for already marked or un
 
 ### Tag a module
 
-> Tag a specified module from the current list of modules in Le Tracker with descriptions
+> `tag {module_code} /tags {tag_1}[, {tag_2}[, {tag_3}, ...]]]`
 
-Format: `tag {module_code} /tags {tag_1}[, {tag_2}[, {tag_3}, ...]]]`
+Tag a specified module.
 
-- `module_code` must belong to an existing module
-- `tag_1, tag_2, ...` must be of correct format
+- <span style="color:#e46c0a">`module_code`</span> : The code of the module to be tagged
+  - Must belong to an existing module in Le Tracker (:exclamation:Module code matching is case sensitive)
+- <span style="color:#e46c0a">`tag_1, tag_2, ...`</span> : The tags to be applied to the module
+  - Must be valid tags (refer to [Argument Formats](#argument-formats) for more information  on the "Tag" format)
```

Comment:
Perhaps use `Refer to [Argument Formats](#argument-formats) for the "Tag" format`.

"Must be valid tags" was from the unfinalised standardisation of the user guide. I felt that it was redundant and just added more "noise" to the description.

This issue is repeated several times in your edit.